src/server/userDisconnected.js

```diff
@@ -0,0 +1,12 @@
+const {
```

Comment:
You are using tabs here while in other files you are using spaces for indentation